code generators/de.fhdo.lemma.ddd/.classpath

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this Eclipse-generated file because you did not change anything in the DDD code generator, Iguess?